support/http/logging_middleware.go

```diff
@@ -10,6 +11,11 @@ import (
 	"github.com/stellar/go/support/log"
 )
 
+// Options allow the middleware logger to accept additional information.
+type Options struct {
+	extraHeaders []string
```

Comment:
❗ The extra headers needs to be exported for users of this package to be able to set it. Capitalize identifiers to make them accessible outside the current package.
```suggestion
	ExtraHeaders []string
```

Reply:
Good suggestion.